cdap-docs/tools/cdap-default/cdap-default-deprecated.xml

```diff
@@ -23,15 +23,15 @@
     <name>app.bind.address</name>
     <value>0.0.0.0</value>
     <description>
-      App Fabric service bind address (deprecated: use master.services.bind.address instead)
+      App Fabric service bind address (deprecated: replaced with ${master.services.bind.address})
```

Comment:
nope, didn't mean for the ``@link`` to be used here. ``${foo}`` should be used here. I'm just talking about consistency in the wording.